### ISSUE TITLE
The "verbose mode" setting could use some description text so people know what it does

### ISSUE THREAD

**Issue by Cris-Edmundson** [futo-org/voice-input#21]
Title. It would just make the ui/ux a little more put together for end users

**Comment by jmwt1839** [comment]
It is exactly what I was trying to work out myself as well. It seems that there is no trace of any explanation in the help or anywhere else.

**Comment by jmwt1839** [comment]
Thank you for the clarification.

**Comment by milkey-mouse** [comment]
I sort of like verbose mode, especially with the larger models. The wait for transcription seems shorter, and I'm less worried the app has crashed, if I see each step complete.

**Comment by Pandapip1** [comment]
I agree with the 'it feels shorter,' even though it's almost certainly just placebo effect. The placebo effect does make things feel faster for the end-user!

**Comment by David-Prock** [comment]
> It is exactly what I was trying to work out myself as well. It seems that there is no trace of any explanation in the help or anywhere else.
> 
> Does anyone know what verbose mode does?

I notice it does more of punctuation and capitalization.
Just hit the test and say one word like hello and then turn it off and do it again and you'll see.

**Comment by e-t-l** [comment]
> It changes the status text to say the stage of model inference instead of just "Processing..." when running the model. It was useful initially for debugging performance. It will probably be removed at some point.

This is true, and undoubtedly the intended purpose, therefore...

> I notice it does more of punctuation and capitalization. Just hit the test and say one word like hello and then turn it off and do it again and you'll see.

This seems like a bug, then. I tested and confirmed that it does add more frequent punctuation. Any idea why/how to fix that?
